src/api/transaction/submit.js

```diff
@@ -5,6 +5,16 @@ const validate = utils.common.validate;
 const Request = utils.common.core.Request;
 const convertErrors = utils.common.convertErrors;
 
+function convertSubmitErrors(callback) {
+  return function(error, data) {
+    if (data.engineResult !== 'tesSUCCESS') {
```

Comment:
What's an example of a code that is not tesSUCCESS that could be a successfully submitted transaction?